app/components/conceptsService/concepts.service.js

```diff
@@ -73,10 +85,19 @@ angular
 					result.message = err;
 					return result;
 				}
+				var descriptions = parseDescriptions(localizedConcepts);
+				if(descriptions.length>0) conceptRequest.descriptions;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'conceptRequest'.](https://www.codacy.com/app/openmrs/openmrs-owa-conceptdictionary/file/2339254612/issues/source?bid=3196969&fileBranchId=3209439#l89)